src/pyspiffe/__init__.py

```diff
@@ -0,0 +1 @@
+SPIFFE_SCHEME = 'spiffe'
```

Comment:
This is at top level. I might be missing something but what about setting this constant on the spiffe_id package?